src/context.ts

```diff
@@ -8,16 +9,17 @@ import { createEntityService } from './entity/service'
 
 interface IO {
   database: Db
+  didResolver: DidResolver
 }
```

Comment:
Not sure `didResolver` should be part of an `IO` object?

Reply:
Good point. Why do you think so? :)

Reply:
I see. I feel it might be considered as an input to our system. I don't see it as a significant part of our domain. I did it as an experiment, and it actually simplifies mocking in `server.test`. However, I'm happy to reconsider that in the future.